docs/DA-303: Update SDK examples (#180)
PolymeshAssociation/technical-content: content/04-distribute/distribute-sdk.mdx

    <MultiCodeBlock>

    ```typescript
    const asset: Asset = await apiCeo.assets.getAsset({
        "ticker" :"ACME"
    });
    ```

    ```javascript
    const asset = await apiCeo.assets.getAsset({
        "ticker" :"ACME"
    });
    ```

    </MultiCodeBlock>
* `acme` is the owner of the ACME token. `acme` will also be the token's primary issuance agent;
* Alice has an account, `alice`, that represents her as an individual;
* Alice the individual also can [instantiate](https://github.com/PolymathNetwork/polymesh-sdk/wiki/api.client.Polymesh.Polymesh#connect) a Polymesh client whenever she needs to act with:
    <MultiCodeBlock>

    ```typescript
    const signingManagerAlice: LocalSigningManager = await LocalSigningManager.create({
        "accounts": [{
            "mnemonic": "word1 word2 ..."
        }]
    });

    const apiAlice: Polymesh = await Polymesh.connect({

>         "nodeUrl": "wss://testnet-rpc.polymesh.live", // For instance, or the proper network

Reviewer: ```suggestion
        "nodeUrl": "wss://testnet-rpc.polymesh.live", // or your preferred node
```

Its not just the network, but what node you want to trust. There are many valid values for any network
